test/fixtures/users.yml

```diff
@@ -10,3 +10,13 @@ tokenuser:
   access_token: <%= SecureRandom.hex(20) %>
   github_login: 'tokenuser'
   personal_access_token: 'deadbeef'
+
+userwithunread:
+  github_id: 92873
+  access_token: <%= SecureRandom.hex(20) %>
+  github_login: 'userwithunread'
+
+morty:
```

Comment:
I added some new characters because I didn't want to break anybody's mental model of the existing test fixtures.